onnx_tf/handlers/backend/gru.py

```diff
@@ -173,9 +305,7 @@ def _common(cls, node, **kwargs):
       h_fw = tf.expand_dims(state_fw, 0)
       h_bw = tf.expand_dims(state_bw, 0)
       h = tf.concat((h_fw, h_bw), axis=0)
-      output_fw = tf.expand_dims(output_fw, 1)
-      output_bw = tf.expand_dims(output_bw, 1)
-      output = tf.concat((output_fw, output_bw), axis=1)
+      output = tf.concat((output_fw, output_bw), axis=-1)
```

Comment:
This return format has kept subsequent GRU cells from taking on a `GRUCellWithLinearBeforeReset`s output. 

`ValueError: Dimension must be 3 but is 4 for 'transpose_41' (op: 'Transpose') with input shapes: [?,8,2048], [4].`

The old formatting worked instead.